code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -251,19 +251,19 @@
 		var/remove_amount = round(min(fuel_list[gas_id], fuel_consumption), 0.01)
 		internal_fusion.adjust_moles(gas_id, -remove_amount)
 		delta_fuel_list[gas_id] -= remove_amount
-	for(var/gas_id in fuel.primary_products)
-		var/add_amount = round(fuel_consumption * 0.5, 0.01)
-		internal_fusion.adjust_moles(gas_id, add_amount)
-		delta_fuel_list[gas_id] += add_amount
 
 	if(power_level < 1)
 		return // can't produce any gases, don't need to continue
 
 	// Each recipe provides a tier list of six output gases.
 	// Which gases are produced depend on what the fusion level is.
 	var/list/tier = fuel.secondary_products
-	moderator_internal.adjust_moles(tier[power_level], round(scaled_production, 0.01)) // gases on the same tier are produced at normal rate
-	delta_mod_list[tier[power_level]] += round(scaled_production, 0.01)
+	var/add_remove_amount = round(scaled_production, 0.01) // gases on the same tier are produced at normal rate
+	moderator_internal.adjust_moles(tier[power_level], add_remove_amount)
+	delta_mod_list[tier[power_level]] += add_remove_amount
+	for(var/gas_id in fuel.primary_products)
+		internal_fusion.adjust_moles(gas_id, add_remove_amount)
+		delta_fuel_list[gas_id] += add_remove_amount
```

Comment:
This section needs to be before the `power_level < 1` check or it doesn't work at level 0

Reply:
i dont think it does at level 0, even if it does it would be very little anyway

Reply:
ill just change it then